src/components/OrderOptions/styled.js

```diff
@@ -0,0 +1,20 @@
+import styled from 'styled-components';
+
+const StyledOrderOptions = styled.div`
+	display: flex;
+	justify-content: center;
+	width: 80vw;
+	margin: 0.5rem;
+	padding: 0.75rem;
+	border: 2px solid burlywood;
+	border-radius: 10px;
+	label {
```

Comment:
I would use 3 Styled Components, 1 for the Div , input and Label.

Reply:
Good idea. I will implement it during the Re-Design process.